src/state_calc.py

```
@@ -0,0 +1,849 @@
+                final_states.append("C" * len(self.__all_cyclins))
+                state_score = self.__calculate_state_scores(all_cyclin_states[-1]) + self.__calculate_state_scores(
+                    all_cyclin_states[-2]
+                )
+            else:
+                curr_final_state = all_cyclin_states[-1]
+                final_states.append("".join(map(str, curr_final_state)))
+                state_score = self.__calculate_state_scores(curr_final_state)
+
+            state_scores_dict["".join(map(str, start_state))] = state_score
+            if self.__view_state_table:
+                self.print_state_table(all_cyclin_states, update_sequence)
```

Pylint (3.3.7)

[error] Instance of 'CellCycleStateCalculation' has no 'print_state_table' member

(E1101)